x/emissions/migrations/v3/migrations.go

```diff
@@ -0,0 +1,18 @@
+package v3
+
+import (
+	"fmt"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
+// An example MigrateStore function that might be run in the migration
+// handler.
+func MigrateStore(ctx sdk.Context) error {
```

Comment:
v3 needed?

Reply:
needed for integration testing upgrade flow